src/renderer/redux/actions/user.js

```diff
@@ -49,6 +66,7 @@ export function doAuthenticate() {
         });
         dispatch(doRewardList());
         dispatch(doFetchInviteStatus());
+        dispatch(doInstallNew());
```

Comment:
Since we aren't storing this in the store, we don't need to wrap this call in a `dispatch`. You can just call `doInstallNew()` after.